### PR TITLE
S2U-14 Samigo: Several improvements related to question cancellation 

### PR DESCRIPTION
**Pull request by stetsche**
Thanks to great QA work from @jesusmmp some issues related to cancelling questions in samigo were identified. This PR should fix all of them. Additionally I disabled the input's for score values of cancelled questions.

https://sakaiproject.atlassian.net/browse/S2U-14

### CODE REVIEW

**Comment by stetsche** [comment]
I split each issue into a separate commit to ease the review process, but they can be squashed when merging.

**Review by jesusmmp — APPROVED** [review]
LGTM!

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/CommonMessages_eu.properties [hunk 1]
```diff
@@ -42,6 +42,7 @@ cancel_question=Ezabatu galdera
 cancel_question_reduce_total=Murriztu puntuazio osoa
 cancel_question_distribute_points=Modu berean banatu puntuak
 cancel_question_cancel=Ez ezabatu galdera
+cancel_question_cancelled=Cancelled question
```
Comment (mpellicer) [inline samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/CommonMessages_eu.properties:45]: @PABLOSANROMAN could you please contribute a translation? Thanks!

**Review by mpellicer — APPROVED** [review]
Very nice! This will be testable on Monday.

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/CommonMessages_eu.properties (hunk 1 above)
Comment (PabloSanRoman) [reply samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/CommonMessages_eu.properties:45]: Hi @mpellicer  

cancel_question_cancelled=Galdera baliogabetua

Comment (mpellicer) [reply samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/CommonMessages_eu.properties:45]: Fixed, thanks a ton!

**Pull request merged**